spec/controllers/proposals_controller_spec.rb

```diff
@@ -194,6 +194,7 @@
       es_execute_with_retries 3 do
         get :download, text: "Work Order", format: "csv"
         expect(response.body).to include "Work Order 29"
+        expect(response.body).to include proposals.last.client_data.approving_official.display_name
```

Comment:
I feel like I'd want to see an email address here -- names are not unique and difficult if someone wants to, say, follow up with XYZ person about something based on a CSV download. Or did we hear otherwise?